### ISSUE TITLE
Open Question: Should 'meta' be changed to 'ctrl'?

### ISSUE THREAD

**Issue by fresheneesz** [fresheneesz/keysight#3]

**Comment by circuitrider** [comment]
Nah, meta seems to be common parlance for the behavior described as far as I'm aware.

**Comment by fresheneesz** [comment]
This seems to suggest that meta is a completely different key in its own right: http://askubuntu.com/questions/19558/what-are-the-meta-super-and-hyper-keys

**Comment by circuitrider** [comment]
Yeah, but as they say "The Meta key is not found on modern keyboards."

I'd defer to the common usage of it instead of the perhaps more technically correct, but obsolete, usage of it.

**Comment by circuitrider** [comment]
Sorry, meaning to say, that in the context of the ctrl key modifier I've always seen it referred to as meta. And since the only possible conflict is obsolete, diverging from the established usage of the term may add to confusion when referencing external resources.

**Comment by fresheneesz** [comment]
KeyboardEvents still have an `isMeta` property. Maybe that's obsolete tho.
